File: src/main/java/com/summer/lijiahao/utils/OpenApiAction.java
package com.summer.lijiahao.utils;

import com.intellij.ide.BrowserUtil;
import com.intellij.openapi.actionSystem.AnActionEvent;
import com.summer.lijiahao.abs.AbstractAnAction;

public class OpenApiAction extends AbstractAnAction {
    private final static String URL = "https://nccdev.yonyou.com/openapi/doc";

    @Override
    public void doAction(AnActionEvent event) {
        BrowserUtil.open(URL);
    }
}

File: src/main/kotlin/com/summer/lijiahao/nccdevtool/menu/action/OpenApiAction.kt
package com.summer.lijiahao.nccdevtool.menu.action

import com.intellij.ide.BrowserUtil
import com.intellij.openapi.actionSystem.AnActionEvent
import com.summer.lijiahao.nccdevtool.base.action.AbstractAnAction

class OpenApiAction : AbstractAnAction() {

    private val URL = "https://nccdev.yonyou.com/openapi/doc"

    override fun doAction(event: AnActionEvent) {
        BrowserUtil.open(URL)
    }
}
